frontend/context/StakingContractDetailsProvider.tsx

```diff
@@ -132,7 +132,17 @@ export const StakingContractDetailsProvider = ({
   children,
 }: PropsWithChildren) => {
   const [isPaused, setIsPaused] = useState(false);
-  const serviceId = useServiceId();
+  const {
+    selectedService,
+    selectedAgentConfig,
+    isFetched: isLoaded,
+  } = useServices();
+  const serviceConfigId =
+    isLoaded && selectedService ? selectedService?.service_config_id : '';
```

Comment:
`useService` handles nil now, shouldn't need this moving forward

```suggestion
    selectedService?.service_config_id;
```